src/router/guards/index.js

```diff
@@ -0,0 +1,5 @@
+import userStore from "../../storage/user.js";
+
+export const isAuthorized = () => {
+  return userStore.isAuthorized();
+};
```

Comment:
**Code Structure Issue**: Direct import and use of userStore in guards can lead to tight coupling and makes unit testing difficult. <br> **Fix**: Inject userStore as a dependency to reduce coupling and improve testability. <br> **Code Suggestion**: 
 ```
 -import userStore from "../../storage/user.js";
 +import { getUserStore } from "../../storage/user.js";
 +
 +export const isAuthorized = (userStore = getUserStore()) => {
 +  return userStore.isAuthorized();
 +};
 ```